Cargo.toml

```diff
@@ -31,10 +31,11 @@ opt-level = 1 # required for the tail calls in inflate to optimize
 
 [workspace.dependencies]
 libloading = "0.8.1"
-libz-sys = { version = "1.1.12", default-features = false, features = ["zlib-ng"] } # use libz-ng in libz compat mode
+libz-sys = { version = "1.1.19", default-features = false, features = ["zlib-ng"] } # use libz-ng in libz compat mode
```

Comment:
this release fixes a bug with a missing file that caused issues for the s390x target https://github.com/rust-lang/libz-sys/pull/205